### PR TITLE
Add fingerprints to Stanage SSH section.

### PR DESCRIPTION
**Pull request by jkwmoore**

### CODE REVIEW

**Comment by jkwmoore** [comment]
Eventually this will be needed in the FAQ section too. @willfurnass please merge when happy.

**Review by willfurnass — APPROVED** [review]

**Comment by willfurnass** [comment]
Should we say the "SSH fingerprint will be _one_ of the following"?

**Comment by jkwmoore** [comment]
I think we can clarify that checking that one of them is matching is sufficient? Technically it does have all of them.

File: stanagepilot/stanage-connecting-ssh.rst [hunk 1]
```diff
@@ -23,7 +23,21 @@ log in to a cluster: ::
...
 
-Launch an interactive session on Stanage using the following command: ::
+When connecting for the first time you should make sure that the SSH fingerprints are correct.
+The RSA, ECDSA and ED25519 fingerprints for Stanage's login nodes are: ::
+
+    SHA256:mFfJmZHH0SUogoUhTtlatoZLEacfGAlj0cTrnInO5z0 (RSA)
+    SHA256:4HdvK3C1KDm+JG1TzxQKxezMz5ojEORynHUqF9tQfoI (ECDSA)
+    SHA256:aaTv+0TEc0nj7WR2ZuBYWFDD+QqzOKJpMjEFKBx6pQU (ED25519)
+
+Your terminal will show one of these fingerprints by default. If any one of these fingerprints matches you should continue.
```
Comment (willfurnass) [inline stanagepilot/stanage-connecting-ssh.rst:33]:
```suggestion
Your SSH client will show one of these fingerprints by default. If any one of these fingerprints matches you should continue. 
```

File: stanagepilot/stanage-connecting-ssh.rst [hunk 2]
```diff
@@ -23,7 +23,21 @@ log in to a cluster: ::
 
     ssh -X $USER@stanage-login1.shef.ac.uk
 
-Launch an interactive session on Stanage using the following command: ::
+When connecting for the first time you should make sure that the SSH fingerprints are correct.
```
Comment (willfurnass) [inline stanagepilot/stanage-connecting-ssh.rst:26]:
```suggestion
When connecting for the first time you should make sure that the SSH 'fingerprint' is correct.
```

**Pull request merged**